### PR TITLE
Main  kk team project optimization

### PR DESCRIPTION
**Pull request by KrystianKjjk**

### CODE REVIEW

File: frontend/src/components/FindSection/FindSection.tsx [hunk 1]
```diff
@@ -1,143 +0,0 @@
...
-import { Backdrop, Fade, Modal, MuiThemeProvider } from '@material-ui/core'
-
-import BaseService from '../../app/baseService'
-import SectionsService from '../../api/sections.service'
-import ReusableTable from '../ReusableTable'
-import SearchInput from '../SearchInput'
-
-import styles from './FindSection.module.css'
-import { mainTheme } from '../../theme/customMaterialTheme'
-
```
Comment (robert-czwartosz) [inline frontend/src/components/FindSection/FindSection.tsx:11]: why this component was removed?

**Review by robert-czwartosz — COMMENTED** [review]
Great job! :+1: 
There are a few issues to solve.

File: frontend/src/components/FindSection/FindSection.tsx (hunk 1 above)
Comment (KrystianKjjk) [reply frontend/src/components/FindSection/FindSection.tsx:11]: Name has been replaced to 'FindProject'

File: api/Src/Services/TeamProjectService.ts [hunk 2]
```diff
@@ -30,10 +25,21 @@ export default class TeamProjectService {
     return this.getTeamProjectsByTeamId(team._id)
   }
 
-  getTeamProjects = async (): Promise<
-    (TeamProject & mongoose.Document<TeamProject>)[]
-  > => {
-    return this.teamProjectRepository.getAll()
+  //fajnie by było przeprowadzic te filtracje jeszcze ana bazie ... ale jak ?
```
[the remark this answers is not in the document]
Comment (KrystianKjjk) [reply api/Src/Services/TeamProjectService.ts]: done

File: api/.prettierignore [hunk 3]
```diff
@@ -0,0 +1,5 @@
+
+./CreateContainer
+
+
+./CreateContainer.ts
```
[the remark this answers is not in the document]
Comment (KrystianKjjk) [reply api/.prettierignore]: done

File: .prettierignore [hunk 4]
```diff
@@ -0,0 +1,7 @@
+
+./api/CreateContainer
+
+
+./api/CreateContainer.ts
+
+CreateContainer.ts
```
[the remark this answers is not in the document]
Comment (KrystianKjjk) [reply .prettierignore]: done

File: frontend/src/api/ReferenceProject.api.ts [hunk 5]
```diff
@@ -84,3 +42,22 @@ export const addRefProject = async (project: any) => {
...
+
+export interface Project {
+  _id: string
+  sectionId: Section
+  projectName: string
+  projectUrl: string
+  description: string
+  createdAt: string
+  updatedAt: string
+}
```
[the remark this answers is not in the document]
Comment (KrystianKjjk) [reply frontend/src/api/ReferenceProject.api.ts:63]: Im not sure about that, now I leave It here, and later we need to consider how to manage types

**Pull request merged**